lib/rules/max-line-length/index.js

```diff
@@ -34,7 +34,11 @@ const rule = function (maxLength, options) {
 
     // Collapse all urls into something nice and short,
     // so they do not throw the game
-    const rootString = root.toString().replace(/url\(.*\)/ig, "url()")
+    const rootString = root.toString()
+    // Replace `url(something)` with `url()`
+    .replace(/url\(.*\)/ig, "url()")
+    // Replace `@import "something"` with `@import ""`
+    .replace(/\@import\s+['"].*['"]/ig, "@import ")
```

Comment:
Thanks @davidtheclark, I think what I have is OK (for my sanity I use http://regexr.com/ to test/explain regexes). There is probably room for improvement as I'm not the greatest regex architect.

Using `\@import\s+['"].*['"]` and replacing with `@import `

With this string: `@import 'svg-something<id="horse">';`

Results in `@import ;` (There is a single space between the `t` and `;`)

I've added a test to accept that the length of this `@import 'svg-something<id=\"horse\">' projection;` string is 20 characters or under, stripping `'svg-something<id=\"horse\">'` leaves `@import  projection;` which is exactly 20 characters in length.

I've also added a failing test using `@import 'svg-something<id=\"horse\">' screens, tv;` which once stripping `'svg-something<id=\"horse\">'` leaves `@import screens, tv;` which is 21 characters in length.